quickstarts/lighttpd/config.yml

```diff
@@ -0,0 +1,41 @@
+
+slug: lighttpd
+description: |
+  ## Comprehensive monitoring for your lighttpd server
+
+  Lighttpd is an open-source web server designed for high-performance situations while staying standards-compliant, secure, and adaptable.
+
+  ### Why monitor your lighttpd server?
+
+  Lighttpd monitoring provides important information to help you construct a complete picture of your web server's performance. This performance data includes uptime, network in bytes and packets, number of connections, and more.
+
+  ### What’s included?
+
+  When you monitor your lighttpd server with New Relic, you get immediate full-stack observability, including:
+  - Be alerted on things like average number of connections, average number of requests and high memory usage.
+  - Use dashboards to monitor crucial performance metrics, such as uptime, connection state handle request, connection state still alive, networks, requests per second, and more.
+
+summary: |
+   With our infrastructure agent and Flex configuration setup, you can monitor and validate your lighttpd processes. Right out of the box, analyze and capture your server's performance data.
```

Comment:
"lighttpd processes. Right out of the box," -> "lighttpd processes right out of the box:"